tools/eslint/package.json

```diff
@@ -5,6 +5,9 @@
   "main": "index.js",
   "repository": "Seneca-CDOT/telescope",
   "license": "BSD-2-Clause",
+  "peerDependencies": {
+    "eslint": "7.32.0"
+  },
```

Comment:
Wow, I didn't know there was a `peerDependencies` key we could use. I've only ever seen `devDependencies` and `dependencies`